src/Transports.AspNetCore/Transports.AspNetCore.csproj

```diff
@@ -16,10 +16,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="2.0.0-alpha-864" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.1" />
+    <PackageReference Include="GraphQL" Version="[2.0.0, 2.1)" />
```

Comment:
I actually think it's better to not specify an upper limit unless a higher version has been released and there are known incompatibilities. 

Otherwise, when GraphQL 2.1 is released, people will have problems updating unless new version of these packages are released at the same time. It will prevent people from upgrading even if there are no compatibility issues. I've seen this happen with other projects and it's really annoying (e.g., https://github.com/openstacknetsdk/openstack.net/issues/656).

Reply:
I suppose the upper limit should be < 3.0.

Reply:
That would be better, but could still cause issues with GraphQL 3.0, even if it's not incompatible. It doesn't look like Microsoft specifies maximum version numbers for dependencies on their NuGet packages.

Reply:
I support dropping the range.  If there are known compatibility issues I can see adding one in, but not specifying it gives flexibility on updating.

Reply:
dropping it